packages/audio-block/src/components/ItemToolbar.tsx

```diff
@@ -0,0 +1,132 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    ActionMenu,
+    Flyout,
+    IconArrowCircleUp20,
+    IconDotsHorizontal16,
+    IconImageStack20,
+    IconTrashBin16,
+    IconTrashBin20,
+    MenuItemContentSize,
+    Tooltip,
+    TooltipPosition,
+} from '@frontify/fondue';
+import { ItemToolbarProps } from '../types';
+
+import { useState } from 'react';
+
+export const ItemToolbar = ({ onRemoveAsset, onUploadClick, onAssetChooseClick }: ItemToolbarProps) => {
+    const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+
+    return (
+        <div data-test-id="item-toolbar" className="tw-absolute tw-z-10 tw-right-0 tw-invisible group-hover:tw-visible">
+            <div className="tw-bg-white tw-text-box-selected-inverse tw-flex tw-flex-shrink-0 tw-gap-[2px] tw-px-[1px] tw-spacing tw-items-center tw-h-[28px] tw-self-start tw-border tw-border-box-selected-inverse tw-rounded-[4px]">
```

Comment:
You probably copied this from other packages but I recently got the feedback to use tailwind values wherever possible. E.g instead of `tw-rounded-[4px]` use `tw-rounded`. Instead of `tw-h-[28px]` use `tw-h-7`

Reply:
removed all fixed values, [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/6dd8c43fc1d95798e8c73fa544df8777dfbe8acd)